src/components/Span/SpanDetailsPanel.tsx

```diff
@@ -258,94 +261,126 @@ export const SpanDetailPanel = ({
   }, [result, debouncedSearch]);
 
   return (
-    <div className="z-10">
-      <div className="overflow-hidden text-sm">
-        <h2 className="uppercase">Span Details</h2>
-        <input type="text" placeholder="Search" value={search} onChange={(e) => setSearch(e.target.value)} />
-        <table className="w-full">
+    <div className="z-10 overflow-hidden text-sm">
+      <div className="px-2 py-2">
+        <div className="flex flex-col gap-4 items-start justify-between py-4 px-2 mx-4">
+          <div className="flex items-center justify-between gap-2 w-full">
+            <span className="uppercase text-lg font-light">Span Details</span>
+            <IconButton
+              size="xxl"
+              name="times"
+              variant="secondary"
+              aria-label="Close"
+              onClick={onClose}
+              className="w-4 h-4"
+            />
+          </div>
+          <Input
+            className="w-full"
+            type="text"
+            placeholder="Search details"
+            value={search}
+            onChange={(e) => setSearch(e.currentTarget.value)}
+            suffix={
+              search ? (
+                <IconButton name="times" variant="secondary" aria-label="Clear" onClick={() => setSearch('')} />
+              ) : (
+                <IconButton name="search" variant="secondary" aria-label="Search" />
+              )
+            }
+          />
+        </div>
+      </div>
+      <div className="px-4">
+        <table className="w-full text-[0.8rem]">
           <tbody>
             {basicSpanData.map((item, index) => (
               <tr key={item.key} className={rowClassName(index)}>
-                <td className="font-semibold text-gray-700 dark:text-gray-300 border-r border-gray-300 dark:border-gray-600 w-1/3 mx-4">
-                  <span className="px-2 py-2">{item.key}</span>{' '}
-                  {/* TODO: padding & margins are overriden to 0 by the global CSS and it is not possible to set it on the td tag */}
+                <td className="font-regular text-gray-700 dark:text-gray-300  w-1/3 mx-4">
+                  <span className="px-2">{item.key}</span>{' '}
```

Comment:
Add a `white-space: nowrap;` to this.
So something like 

<img width="377" height="63" alt="Image" src="https://github.com/user-attachments/assets/e9270b2a-34ce-4e5f-a6de-b59e6b2a768e" /> 

isn't split.